package.json

```diff
@@ -18,7 +18,7 @@
   "main": "app/index.js",
   "repository": "h5bp/generator-server-configs",
   "scripts": {
-    "test": "mocha --timeout 50000"
+    "test": "mocha --timeout 60000"
```

Comment:
> increased tests timeout to let node-sass compile when installing server-configs-node.
Compilation may take some time, on my machine (1 yo MBP), it took more that 50sec.